mail/chrome/communicator/utilityOverlay.dtd

```diff
@@ -2,55 +2,55 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
-<!ENTITY fileMenu.label     "Plik">
+<!ENTITY fileMenu.label "Plik">
 <!ENTITY fileMenu.accesskey "P">
-<!ENTITY newMenu.label      "Utwórz">
-<!ENTITY newMenu.accesskey  "U">
+<!ENTITY newMenu.label "Utwórz">
+<!ENTITY newMenu.accesskey "U">
 
-<!ENTITY editMenu.label     "Edycja">
+<!ENTITY editMenu.label "Edycja">
 <!ENTITY editMenu.accesskey "E">
-<!ENTITY undoCmd.label      "Cofnij">
-<!ENTITY undoCmd.accesskey  "C">
-<!ENTITY undoCmd.key        "Z">
-<!ENTITY redoCmd.label      "Przywróć">
-<!ENTITY redoCmd.accesskey  "r">
-<!ENTITY redoCmd.key        "Y">
-<!ENTITY cutCmd.label       "Wytnij">
-<!ENTITY cutCmd.accesskey   "t">
-<!ENTITY cutCmd.key         "X">
-<!ENTITY copyCmd.label      "Kopiuj">
-<!ENTITY copyCmd.accesskey  "K">
-<!ENTITY copyCmd.key        "C">
-<!ENTITY pasteCmd.label     "Wklej">
+<!ENTITY undoCmd.label "Cofnij">
+<!ENTITY undoCmd.accesskey "C">
+<!ENTITY undoCmd.key "Z">
+<!ENTITY redoCmd.label "Przywróć">
+<!ENTITY redoCmd.accesskey "r">
+<!ENTITY redoCmd.key "Y">
+<!ENTITY cutCmd.label "Wytnij">
+<!ENTITY cutCmd.accesskey "t">
+<!ENTITY cutCmd.key "X">
+<!ENTITY copyCmd.label "Kopiuj">
+<!ENTITY copyCmd.accesskey "K">
+<!ENTITY copyCmd.key "C">
+<!ENTITY pasteCmd.label "Wklej">
 <!ENTITY pasteCmd.accesskey "e">
-<!ENTITY pasteCmd.key       "V">
-<!ENTITY deleteCmd.label        "Usuń">
-<!ENTITY deleteCmd.accesskey    "U">
-<!ENTITY selectAllCmd.label     "Zaznacz wszystko">
+<!ENTITY pasteCmd.key "V">
+<!ENTITY deleteCmd.label "Usuń">
+<!ENTITY deleteCmd.accesskey "U">
+<!ENTITY selectAllCmd.label "Zaznacz wszystko">
 <!ENTITY selectAllCmd.accesskey "a">
-<!ENTITY selectAllCmd.key       "A">
+<!ENTITY selectAllCmd.key "A">
 <!ENTITY preferencesCmd2.label "Opcje">
 <!ENTITY preferencesCmd2.accesskey "O">
 <!ENTITY preferencesCmdUnix.label "Preferencje">
 <!ENTITY preferencesCmdUnix.accesskey "P">
 
-<!ENTITY viewMenu.label             "Widok">
-<!ENTITY viewMenu.accesskey         "W">
-<!ENTITY viewToolbarsMenu.label     "Paski narzędzi">
+<!ENTITY viewMenu.label "Widok">
+<!ENTITY viewMenu.accesskey "W">
+<!ENTITY viewToolbarsMenu.label "Paski narzędzi">
 <!ENTITY viewToolbarsMenu.accesskey "n">
-<!ENTITY showTaskbarCmd.label       "Pasek stanu">
-<!ENTITY showTaskbarCmd.accesskey   "e">
+<!ENTITY showTaskbarCmd.label"Pasek stanu">
```

Comment:
Fixed in https://hg.mozilla.org/l10n-central/pl/rev/5d99d3cd7b36 and https://github.com/aviarypl/mozilla-l10n-central-pl/commit/7c9183f13498e1112bd2186e0c4445dccbdb88b8